### PR TITLE
Guess game PR

### PR DESCRIPTION
**Pull request by Valbjornpdx**

### CODE REVIEW

File: client/src/GuessGame/DataMap.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types';
```
Comment (nayamairena) [inline client/src/GuessGame/DataMap.jsx]: Great job at implementing this game, I don't fully understand what is happening in this file but its cool to see all the logic and functionality needed to execute this game.

File: client/src/GuessGame/DataMap.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import React from 'react';
+import Datamaps from 'datamaps';
```
Comment (nayamairena) [inline client/src/GuessGame/DataMap.jsx]: Using external js resources like this is really smart!

File: client/src/GuessGame/GuessGame.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+  // Using random integer get the random country
+  const getRandCountry = () => {
+    const countryNum = getRandInteger(0, maxScore - 1);
+    setCurrCountry(countryList[countryNum]);
+    console.log('Random country: ', currCountry, countryList[countryNum]);
+  };
+
+  //Get the distance via euclidean maths
+  const distance = (srcLat, srcLong) => {
```
Comment (nayamairena) [inline client/src/GuessGame/GuessGame.js]: I enjoy learning about all the logic and math that goes into building this game.

File: client/src/GuessGame/GuessGame.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+      // console.log(
+      //   ' Lat: ' + guessCode['latitude'] + ' Long: ' + guessCode['longitude']
+      // );
+      // console.log(dist);
+    }
+  };
+
+  //
+  return (
```
Comment (nayamairena) [inline client/src/GuessGame/GuessGame.js]: Nice execution of utilizing your HTML code into JSX to have it work with React. I personally struggle with integrating previous HTML logic to React but its cool to see it be possible!

**Review by nayamairena — COMMENTED** [review]
Good job at creating this game! I will have to test it out once it gets merged and I add it to the UI I have been working on.

**Comment by AdrikGG** [comment]
Great work! I'll be integrating it in today.

**Pull request merged**